core/arena/internal/_isPlaceholder.js

```diff
@@ -0,0 +1 @@
+export const _isPlaceholder = (a) => a != null && typeof a === 'object' && a['@@functional/placeholder'] === true
```

Comment:
the @@functional/placeholder is ramda specific. do we use it? if not, maybe we could remove it?